SDL/SDLGLGraphicsContext.cpp

```diff
@@ -120,6 +125,11 @@ EGLConfig EGL_FindConfig(int *contextVersion) {
 		EGLint caveat = readConfig(EGL_CONFIG_CAVEAT);
 		int caveatScore = caveat == EGL_NONE ? 100 : (caveat == EGL_NON_CONFORMANT_CONFIG ? 50 : 0);
 
+#ifndef USING_FBDEV
+		EGLint surfaceType = readConfig(EGL_SURFACE_TYPE);
+		int surfaceScore = (surfaceType & EGL_WINDOW_BIT) ? 100 : (caveat == EGL_NON_CONFORMANT_CONFIG ? 50 : 0);
```

Comment:
Hmm, wait, this might accidentally boost non-conformant configs?  Copy/paste mistake?

-[Unknown]

Reply:
Yeah I just took it verbatim from your post :) You're right that this might not be great..